### PR TITLE
Bug 1834734: add integration test for affinity modal

### PR DESCRIPTION
**Pull request by glekner**

### CODE REVIEW

File: frontend/packages/kubevirt-plugin/integration-tests/tests/vm.detail.affinity.scenario.ts [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+  afterAll(() => {
+    deleteResource(testVM);
+  });
+
+  it(
+    'Adds an Affinity, then removes it',
+    async () => {
+      await vm.navigateToDetail();
+      await vm.modalEditAffinity();
+      await click(editAffinityView.addAffinityBtn);
```
Comment (rhrazdil) [inline frontend/packages/kubevirt-plugin/integration-tests/tests/vm.detail.affinity.scenario.ts:28]: could we test that added affinity actually takes an effect?

File: frontend/packages/kubevirt-plugin/integration-tests/tests/vm.detail.affinity.scenario.ts [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+  beforeAll(async () => {
+    createResource(testVM);
+  });
+
+  afterAll(() => {
+    deleteResource(testVM);
+  });
+
+  it(
+    'Adds an Affinity, then removes it',
```
Comment (rhrazdil) [inline frontend/packages/kubevirt-plugin/integration-tests/tests/vm.detail.affinity.scenario.ts:24]: ID(4159)
https://polarion.engineering.redhat.com/polarion/#/project/CNV/workitem?id=CNV-4159

**Comment by glekner** [comment]
added id @rhrazdil

**Pull request merged**